ui/ducks/send/send.js

```diff
@@ -1821,6 +1813,7 @@ export function updateGasPrice(gasPrice) {
       actions.updateGasFees({
         gasPrice,
         transactionType: TRANSACTION_ENVELOPE_TYPES.LEGACY,
+        manuallyEdited: true,
```

Comment:
I mist be misunderstanding something. This makes me think that this is the only method whereby gas can be manually edited, is that correct? That if this is getting called, it's because a manual edit has occurred?

If that's the case, then why are we discarding this `gasPrice`? It looks like it gets ignored in `updateGasFees` if `manuallyEdited` is `true`

Reply:
Yes, you are correct. The if block I added should be moved down.

Reply:
and also...

Reply:
Okay, I have fixed this with 609506ddda

> this is the only method whereby gas can be manually edited, is that correct? That if this is getting called, it's because a manual edit has occurred?

Yes, that is correct.

> If that's the case, then why are we discarding this gasPrice? It looks like it gets ignored in updateGasFees if manuallyEdited is true

Correct, that was a bad mistake. I have fixed this with 609506ddda. Now we don't ignore gasPrices in the action payload if that action is a manual update.

Reply:
Got it, thanks, this makes sense now